command/agent/auth/approle/approle.go

```diff
@@ -21,6 +21,7 @@ type approleMethod struct {
 
 	roleIDFilePath                 string
 	secretIDFilePath               string
+	secretIDLessMode               bool
```

Comment:
Is there a reason you need this bool? Seems like simply checking the path for emptiness should be sufficient.

Reply:
👍 You are right, PR updated